facebookads/objects.py

```diff
@@ -576,9 +576,13 @@ def callback_failure(response):
                 if failure is not None:
                     failure(response)
 
+            relative_url = self.get_node_path()
+            if params.get('fields'):
+                relative_url = '{}?fields={}'.format('/'.join(relative_url), params['fields']),
```

Comment:
Would `self.get_node_path_string()` be preferred to `'/'.join(self.get_node_path()`?